.travis.yml

```diff
@@ -15,12 +15,8 @@ env:
 matrix:
   fast_finish: true
   include:
-    - php: 5.6
-      env: COMPOSER_FLAGS="--prefer-lowest" SYMFONY_DEPRECATIONS_HELPER=weak
```

Comment:
Shouldn't `--prefer-lowest` tests be moved to another test suite?

Reply:
Oh right, good catch.

Reply:
We should make a choice, because here we have two ways to do the same thing
--prefer-lower (so SF 4.4) or SYMFONY_VERSION=4.4

Reply:
I choose prefer-lowest

Reply:
Those are actually two different things. With `--prefer-lowest` we would test against Symfony 4.4.0, but there should also be a test run with the latest 4.4 release. I think we need both.

Reply:
ok, I'll do that 👍🏼